config-controller/internal/controller/policy_controller.go

```diff
@@ -93,59 +118,49 @@ func (r *SecurityPolicyReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 			// remove our finalizer from the list and update it.
 			controllerutil.RemoveFinalizer(policyCR, policyFinalizer)
 			if err := r.K8sClient.Update(ctx, policyCR); err != nil {
-				return ctrl.Result{}, errors.Wrapf(err, "failed to remove finalizer from policy CR %q", policyCR.GetName())
+				return ctrl.Result{}, errors.Wrapf(err, "unable to add finalizer to policy %q", policyCR.GetName())
```

Comment:
I modified this to the correct message in my PR. "failed to remove finalizer ....